splitwise/splitwise.py

```diff
@@ -0,0 +1,42 @@
+from user import User
+from constants import *
+
+class Splitwise:
+    def __init__(self, users_list):
+        self.users_graph = [ [ 0 for _ in range(User.user_count)] for i in range(User.user_count) ]
+        self.users = users_list
+    
+    def get_users_list(self):
+        return self.users
+
+    def add_expense(self, expense):
+        if EQUAL in expense:
+            paid_by = expense[1]
+            amt = expense[2]
+            tot_users = expense[3]
+            owe_users = expense[4:-1]
+            self.add_equal_expense(paid_by, amt, tot_users, owe_users)
+        elif EXACT in expense:
+            #remaining
+            self.add_Exact_Expense()
```

Comment:
why capital Es?

Reply:
That is for the keyword the user will be giving.
Do you mean to say that I should have checked ignoring the letter case by converting it in lower or upper case and then checking it?

Reply:
No. I mean why does the method name contain capital letters when you're not using camelCase?